.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: stable
+    hooks:
+    - id: black
+      language_version: python3.6
```

Comment:
What if you put python3.5 here? Would that fix the issues?

Reply:
Just fixed the issue by using the black from the master branch in the pre-commit hook. `language_version:` here just refers to the python version of the virtual env in which the precommit hook is run. The python version that black supports is configured in pyproject.toml